staticd/static_routes.c

```diff
@@ -760,3 +781,30 @@ void static_route_info_init(struct static_route_info *si)
 {
 	static_path_list_init(&(si->path_list));
 }
+
+
+void static_get_nh_str(struct static_nexthop *nh, char *nexthop, size_t size)
```

Comment:
nexthop2str?  Why are we creating a special function>

Reply:
Actually staticd has a struct static_nexthop, nexthop2str is looking for struct nexthop.
So I have to write new API.